crates/net/network/src/transactions/validation.rs

```diff
@@ -301,14 +149,28 @@ impl FilterAnnouncement for EthMessageFilter {
                 return false
             };
 
-            match self.should_fetch(*ty, hash, *size, &mut tx_types_counter) {
-                ValidationOutcome::Fetch => true,
-                ValidationOutcome::Ignore => false,
-                ValidationOutcome::ReportPeer => {
+            //
+            //  checks if tx type is valid value for this network
+            //
+            let tx_type = match TxType::try_from(*ty) {
```

Comment:
for this we need to find a better solution directly in the NetworkPrimitives abstraction, but this still holds for now, so we can keep this